templates/pro.html

```diff
@@ -0,0 +1,540 @@
+{% extends "_base/base.html" %}
+{% set active_page = "enterprise-support" %}
+
+{% block title %}Ubuntu Proエンタープライズ向けサポート | Ubuntu{% endblock %}
+
+{% block meta_description %}Landscapeは、CanonicalのサービスプログラムであるUbuntu Proで提供されるシステム管理ツールです。Ubuntuマシンの大規模なデプロイをまとめて簡単に管理し、大規模なデスクトップ、サーバー、クラウドのインスタンスのネットワークのサポートにおいてコスト効率を大幅に{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1xmk7zYUzWILnbW3RxGtC9c9stYYEFtKBytETlIgyHYE/edit{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block outer_content %}
+  <div class="p-section">
+    <section class="p-suru--25-75">
+      <div class="row">
+        <div class="col-3">
```

Comment:
Adding a .5 margin-top to the div on line 18 aligns this better with the heading text, we have a custom class for this on u and c.com (`p-image-wrapper`)  
![image](https://github.com/user-attachments/assets/ca3fe43d-8099-4fde-a204-0d5320bbcc40)

vs
![image](https://github.com/user-attachments/assets/2fd129d5-892d-48ed-8e59-a0727f918d21)